### PR TITLE
Add pre-trained model classification script

### PR DESCRIPTION
**Pull request by maraoz**
I downloaded the code and a pre-trained model and wanted to give it a try with an image of my cat, and I couldn't find any code to do that. I wrote a simple script that prints the top 5 predictions of a pre-trained model giving the labels and probabilities, and I think it could be of use to other people trying out the repo.

### CODE REVIEW

**Comment by colesbury** [comment]
Thanks, this looks great. I can can merge it once you sign the CLA.

**Comment by szagoruyko** [comment]
there is a notebook in torch/tutorials https://github.com/torch/tutorials/blob/master/7_imagenet_classification.ipynb and this script https://github.com/torch/tutorials/blob/master/7_imagenet_classification/classify.lua doing the same. resnet will need minimal modifications.
